modules/localization/msf/local_integ/measure_republish_process.cc

```diff
@@ -58,23 +59,23 @@ Status MeasureRepublishProcess::Init(
 
   std::ifstream imu_ant_fin(params.ant_imu_leverarm_file.c_str());
   AINFO << "the ant_imu_leverarm file: "
-            << params.ant_imu_leverarm_file.c_str() << std::endl;
+        << params.ant_imu_leverarm_file.c_str() << std::endl;
```

Comment:
you can remove `std::endl` at line end completely when use apollo log Macro.

Reply:
I have Already done it, please review again.